test/legacy42/jstests/tool/dumprestore_auth2.js

```diff
@@ -108,11 +113,15 @@ var dumpRestoreAuth2 = function(backup_role, restore_role) {
     assert.eq(1, admindb.system.roles.find({role: 'customRole'}).count(), "didn't restore roles");
     assert.eq(2, admindb.system.users.getIndexes().length, "didn't maintain user indexes");
     assert.eq(2, admindb.system.roles.getIndexes().length, "didn't maintain role indexes");
-    assert.eq(
-        1, admindb.system.version.find({_id: "authSchema"}).count(), "didn't restore version");
-    assert.docEq(versionDoc,
-                 admindb.system.version.findOne({_id: "authSchema"}),
-                 "version doc wasn't restored properly");
+    // authSchema document has been removed from server version 8.1.0+, no need to check for it
+    // load isAtLeastVersion from qa tests for the time being.
+    if (!isAtLeastVersion(admindb.version(), "8.1.0")) {
```

Comment:
Could put this into a `adminHasAuthSchemaDocument(client)` function to avoid repeating the `8.1.0`.